models/knownorigin/knownorigin_ethereum_fees.sql

```diff
@@ -0,0 +1,42 @@
+ {{
+  config(
+        alias='fees',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "knownorigin",
+                                    \'["cat"]\') }}')
+}}
+
+SELECT blockchain,
+project,
+version,
+block_time,
+token_id,
+collection,
+platform_fee_amount_raw,
+platform_fee_amount,
+platform_fee_amount_usd,
+CAST(platform_fee_percentage AS DOUBLE) AS platform_fee_percentage,
```

Comment:
if these get cast at the events table, then the cast can likely be removed here. i know a few views on other projects still have the cast, but as long as event table has it, we'll be covered as the view pulls from underlying table